### ISSUE TITLE
Users are banned. is blocked from the group when command is used.

### ISSUE THREAD

**Issue by muhammedfurkan** [0x6c66/Mr-Proper#4]
Users are banned. is blocked from the group when command is used.
it is only necessary to kick when using the kick command. not banned

**Comment by 4m4rOk** [comment]
This is correct. It's not intentional and I think I've fixed it in a local version of the code. Will need to verify and push later.
